secpi/model/dbmodel.py

```diff
@@ -162,12 +162,16 @@ class Param(Base):
     description = Column(Text)
     object_type = Column(Text, nullable=False)
 
-    object_id = Column(Integer, ForeignKey("actions.id"), ForeignKey("notifiers.id"), ForeignKey("sensors.id"))
+    # object_id = Column(Integer, ForeignKey("actions.id"), ForeignKey("notifiers.id"), ForeignKey("sensors.id"))
+    action_id = Column(Integer, ForeignKey("actions.id"))
+    sensor_id = Column(Integer, ForeignKey("sensors.id"))
+    notifier_id = Column(Integer, ForeignKey("notifiers.id"))
 
     def __repr__(self):
         return (
-            f"Param(id={self.id}, object_type={self.object_type}, object_id={self.object_id}, "
+            f"Param(id={self.id}, object_type={self.object_type}, action_id={self.action_id}, sensor_id={self.sensor_id}, notifier_id={self.notifier_id}"
```

Comment:
I think we can do better here. In this manner, the lines - also in the tests! - will not get too long.

```python
    def __repr__(self):
        object_id = None
        if self.action_id is not None:
            object_id = self.action_id
        if self.notifier_id is not None:
            object_id = self.notifier_id
        if self.sensor_id is not None:
            object_id = self.sensor_id
        return (
            f"Param(id={self.id}, object_type={self.object_type}, object_id={self.object_id}, "
            f"key={self.key}, value={self.value})"
        )
```

In this manner, it may even be compatible with the existing test cases.

Reply:
Thank you for using the snippet with ab453e6b. It was slightly wrong, so I fixed it with 0b49dfc5c. Apologies.